### ISSUE TITLE
Create prototype of form error summary pattern

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1021]
## What

Create a working prototype pattern for a form error summary. The pattern should include the following:

- An error summary container at the top of the form, which confirms there has been an issue when submitting the form.
- The summary should contain a list of links containing error messages that link to the individual inputs.
- The page title should be prepended with `Error:`
- Keyboard focus should be moved to the summary
- The summary title should be "There is a problem"
- Continue to show the individual error text next to each input

A page title change should also be documented and included for success messages to prevent AT users from having to navigate the page to find a success flash message.

A similar example can be found in the GDS docs: https://design-system.service.gov.uk/components/error-summary/

## Why

To increase conformance for WCAG 2.1 3.3.1, provide a better experience for AT users and improve overall usability.

3.3.1 success criterion details the above approve here: https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html 
https://www.w3.org/WAI/WCAG21/Techniques/general/G83
https://www.w3.org/WAI/WCAG21/Techniques/general/G139

Details around success messages: 
https://www.w3.org/WAI/WCAG21/Techniques/general/G199

**Comment by jamesjacobs** [comment]
Resolved by #1029
